modules/renter/persist_test.go

```diff
@@ -149,17 +193,6 @@ func TestRenterSaveLoad(t *testing.T) {
 	if err := equalFiles(f3, rt.renter.files[f3.name]); err != nil {
 		t.Fatal(err)
 	}
-
-	// Corrupt a renter file and try to reload it.
-	err = ioutil.WriteFile(filepath.Join(rt.renter.persistDir, "corrupt"+ShareExtension), []byte{1, 2, 3}, 0660)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	err = rt.renter.load()
-	if err == nil {
-		t.Fatal("expected error, got nil")
-	}
```

Comment:
it no longer triggers an error; corrupted .sia files are logged, but don't cause `load` to fail.